WithSecure/withsecure/isolate_device_from_network_action.py

```diff
@@ -8,13 +9,22 @@ class ActionArguments(BaseModel):
     message: str | None = None
 
 
+class RemoteOperationResponse(BaseModel):
+    multistatus: list[Any]
+    transactionId: str
+
+
 class IsolateDeviceFromNetworkAction(DeviceOperationAction):
-    def run(self, arguments: ActionArguments) -> None:
+    def run(self, arguments: ActionArguments) -> RemoteOperationResponse:
         parameters = {}
         if arguments.message:
             parameters["message"] = arguments.message
 
         # execute the operation
-        self._execute_operation_on_device(
+        response = self._execute_operation_on_device(
             operation_name="isolateFromNetwork", target=arguments.target, parameters=parameters
         )
+
+        return RemoteOperationResponse(
+            multistatus=response.get("multistatus"), transactionId=response.get("transactionId")
```

Comment:
As the properties cannot be null
```suggestion
            multistatus=response.get("multistatus", []), transactionId=response["transactionId"]
```